nixos/modules/services/mail/postfix.nix

```diff
@@ -24,7 +24,7 @@ let
     then [ (concatStringsSep ", " (map (s: "reject_rbl_client " + s) cfg.dnsBlacklists)) ]
     else [];
 
-  clientRestrictions = concatStringsSep ", " (clientAccess ++ dnsBl);
+  clientRestrictions = clientAccess ++ dnsBl;
```

Comment:
I don't really get why you do this change. Could you explain it?

Reply:
I think I made a mistake there, as I can't think of why I did that. Thank you 👍

Reply:
The module adds commas inconsistently. Commas seem to be optional in postfix's main.cf so we should either add them for all options that are lists or not.